cz.cvut.edu.comparch.qtrvsim.json

```diff
@@ -0,0 +1,31 @@
+{
+    "app-id" : "cz.cvut.qtrvsim",
+    "runtime" : "org.kde.Platform",
+    "runtime-version" : "6.2",
+    "sdk" : "org.kde.Sdk",
+    "command" : "qtrvsim_gui",
+    "finish-args" : [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/gvfs"
```

Comment:
if you need gvfs access, this is insufficient.

Reply:
not an GTK app.., thanks!